test-app/frontend/package.json

```diff
@@ -17,6 +21,7 @@
     "@types/react-dom": "^17.0.11",
     "aws-amplify": "^4.3.21",
     "bootstrap": "^5.1.3",
+    "mapbox-gl": "^2.9.1",
```

Comment:
bleh

Reply:
lol that should be removed in the latest but I will double check

Reply:
Adding it back for the time being, getting weird import errors that I am not sure how to resolve.